source/createobjects.lua

```diff
@@ -2,7 +2,7 @@ local createobjects = {}
 
 
 function createobjects.CreateObject(intType, intXValue)
--- creates a base and appends it to the garrObjects table

```

Comment:
GROUND looks like a constant (as per style guide) but that is a good driver to move away from globals.